src/HealthCheck/DoctrineConnectionHealthCheck.php

```diff
@@ -51,11 +50,20 @@ public function check(HealthReportInterface $report)
     {
         // Was the entityManager injected? When it is not the project does not use Doctrine ORM
         if (!is_null($this->entityManager)) {
-            // Try a SELECT 1 query on the database to test if it's up and running
             try {
-                $this->entityManager->getConnection()->exec('SELECT 1;');
+                // Get the schema manager and grab the first table to later query on
+                $sm = $this->entityManager->getConnection()->getSchemaManager();
+                $tables = $sm->listTables();
+                if (!empty($tables)) {
+                    $table = reset($tables);
+                    // Perform a light-weight query on the chosen table
+                    $query = 'SELECT * FROM `%s` LIMIT 1';
+                    $this->entityManager->getConnection()->exec(sprintf($query, $table->getName()));
+                    $this->entityManager->getConnection()->close();
```

Comment:
Should this not be outside the `if` statement (so before line 64)? So that we also close the connection if there are 0 tables (should normally not happen, but don't see need to be sloppy in that case).

Reply:
I think it's better if that's handled by Symfony

Reply:
When the query fails, Symfony/Doctrine does not handle garbage collection correctly. Resulting in open database connections (at least for the setup that we use in test/acc/prod). 

Moving the `close` call to the catch block might be the best solution here, as the happy flow would result in Doctrine closing the connection by itself.

@thijskh, I suggest trying this solution on test, as we are unable to reproduce this behavior on our dev machine.